setup.py

```diff
@@ -19,7 +19,7 @@
 import setuptools
 
 REQUIREMENTS = [
-    "qiskit-terra>=0.25.0",
+    "qiskit>=0.25.1",
```

Comment:
This should be 0.44.1

```suggestion
    "qiskit>=0.44.1",
```